translations/en-us.json

```diff
@@ -0,0 +1,161 @@
+{
```

Comment:
This is another thing -- I think they want it to be `en-us` instead of `en`. I kept both, but I think we could safely remove `en` and change the instances to `en-us`.